opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/TracingFilterTest.java

```diff
@@ -243,18 +243,16 @@ public void testAsyncTimeout() throws IOException {
 
         MockSpan mockSpan = mockSpans.get(0);
         Assert.assertEquals("GET", mockSpan.operationName());
-        Assert.assertEquals(6, mockSpan.tags().size());
+        Assert.assertEquals(5, mockSpan.tags().size());
```

Comment:
Why did it change? Is it because of the removal of the error flag?

Reply:
The change is subject of this PR :)